scripts/base/protocols/dns/main.zeek

```diff
@@ -60,9 +60,17 @@ export {
 		## The Recursion Available bit in a response message indicates
 		## that the name server supports recursive queries.
 		RA:            bool               &log &default=F;
-		## A reserved field that is usually zero in
-		## queries and responses.
+		## A reserved field that is usually zero in queries and
```

Comment:
Should we drop the comment as it is not applicable today anymore?

Reply:
Updated comment to match the zeek-docs, let me know if that addresses your concern